src/www/ui/api/Controllers/CustomiseController.php

```diff
@@ -64,4 +64,22 @@ public function getCustomiseData($request, $response, $args)
       return $response->withJson($returnVal->getArray(), $returnVal->getCode());
     }
   }
+
+  /**
+   * Update config data for the admin
+   *
+   * @param  ServerRequestInterface $request
+   * @param  ResponseHelper         $response
+   * @param  array                  $args
+   * @return ResponseHelper
+   */
+
+  public function updateCustomiseData($request, $response, $args)
+  {
+    if (Auth::isAdmin()) {
```

Comment:
If the user is not an admin, no response will be returned failing slim.
Please do it other way round.

See https://github.com/fossology/fossology/pull/2532/files#diff-b1792036fc7f90a6ddc990bd43888841e7ced32722e90481cd47e278d58dca8bR37-R40 for reference.